### PR TITLE
Change baseDC2_5000 to base5000

### LINKED ISSUE #16: include metadata or descriptions for each quantity
As requested by https://github.com/LSSTDESC/descqa/issues/16 it would be useful to include metadata or descriptions for each quantity. 

This can either be done in the yaml file as part of the catalog description, or at a separate place and we modify the base class to read in.

### PR DESCRIPTION
**Pull request by evevkovacs**
> If this pull request is to address specific issues, please add "fix #ISSUE_NUMBER" below so that when this PR is merged, the associated issues will be automatically closed. Always use a short but descriptive title; avoid using "issues/xxx" as title.
fix #16 in desc-help ; change baseDC2_5000 to base5000 to avoid confusion with cosmoDC2

### CODE REVIEW

**Comment by yymao** [comment]
Thanks @evevkovacs. I'll give @JoanneBogart this one to review since she's working on several configs right now, so she can make sure there's no conflicts, etc.

**Comment by yymao** [comment]
I agree, @JoanneBogart. I've updated this PR's merge base to your branch.

File: GCRCatalogs/catalog_configs/base5000_v1.1.1.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 subclass_name: cosmodc2.BaseDC2GalaxyCatalog
 
-catalog_root_dir: /global/cfs/cdirs/lsst/catalogs/baseDC2_5000_v1.1.1
-catalog_filename_template: baseDC2_z_{}_{}_cutout_{}.hdf5
+catalog_root_dir: /global/cfs/cdirs/lsst/shared/xgal/skysim/base5000_v1.1.1
```
Comment (JoanneBogart) [inline GCRCatalogs/catalog_configs/base5000_v1.1.1.yaml:5]: In the new scheme this path needs to be
^/xgal/skysim/base5000_v1.1.1

Comment (evevkovacs) [reply GCRCatalogs/catalog_configs/base5000_v1.1.1.yaml]: I just pushed this change. Should be ready to merge now.

**Comment by evevkovacs** [comment]
@JoanneBogart  Apparently there are conflicts. I tried to do the steps outlined above and got an error:
 git fetch origin
remote: Enumerating objects: 179, done.
remote: Counting objects: 100% (179/179), done.
remote: Compressing objects: 100% (146/146), done.
remote: Total 179 (delta 126), reused 45 (delta 32), pack-reused 0
Receiving objects: 100% (179/179), 32.13 KiB | 8.03 MiB/s, done.
Resolving deltas: 100% (126/126), completed with 3 local objects.
From github.com:LSSTDESC/gcr-catalogs
   698f4df..4489ea4  master                 -> origin/master
 * [new branch]      u/jrbogart/move-to-cfs -> origin/u/jrbogart/move-to-cfs
cori11 GCRCatalogs/catalog_configs> git checkout -b base5000 origin/base5000
fatal: A branch named 'base5000' already exists.
cori11 GCRCatalogs/catalog_configs> git merge u/jrbogart/move-to-cfs
merge: u/jrbogart/move-to-cfs - not something we can merge

Not sure what to do next. Help!

**Comment by JoanneBogart** [comment]
I think the instructions are meant for someone (like me) who already has the u/jrbogart/move-to-cfs branch and needs your branch.   I'll try it.

**Pull request merged**